sections/merge-accounts/burn/BurnInfoBox.tsx

```diff
@@ -0,0 +1,247 @@
+import { FC, useMemo } from 'react';
+import styled from 'styled-components';
+import { Trans, useTranslation } from 'react-i18next';
+import { Svg } from 'react-optimized-image';
+import { useRecoilValue } from 'recoil';
+import useSynthetixQueries from '@synthetixio/queries';
+import { wei } from '@synthetixio/wei';
+
+import ArrowRightIcon from 'assets/svg/app/arrow-right.svg';
+import { formatCurrency } from 'utils/formatters/number';
+import { EXTERNAL_LINKS } from 'constants/links';
+import { FlexDivCentered } from 'styles/common';
+import { CryptoCurrency, Synths } from 'constants/currency';
+import { walletAddressState } from 'store/wallet';
+import { parseSafeWei } from 'utils/parse';
+
+import BarStatsRow from 'sections/staking/components/StakingInfo/BarStatsRow';
+
+import {
+	Title,
+	Subtitle,
+	StyledLink,
+	DataContainer,
+	DataRow,
+	RowTitle,
+	RowValue,
+	ValueContainer,
+	InfoContainer,
+	InfoHeader,
+} from 'sections/staking/components/common';
+
+import { amountToBurnState } from 'store/staking';
+
+import useStakingCalculations from 'sections/staking/hooks/useStakingCalculations';
+
+import {
+	getStakingAmount,
+	getTransferableAmountFromBurn,
+	sanitiseValue,
+} from 'sections/staking/components/helper';
+
+type InfoLayoutProps = {};
+
+const InfoLayout: FC<InfoLayoutProps> = () => {
+	const { t } = useTranslation();
+
+	const {
+		debtBalance,
+		targetCRatio,
+		currentCRatio,
+		transferableCollateral,
+		stakedCollateral,
+		SNXRate,
+		issuableSynths,
+		debtEscrowBalance,
+		collateral,
+	} = useStakingCalculations();
+
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { useSynthsBalancesQuery } = useSynthetixQueries();
+
+	const synthsBalancesQuery = useSynthsBalancesQuery(walletAddress);
+
+	const amountToBurn = useRecoilValue(amountToBurnState);
+
+	const sUSDBalance = synthsBalancesQuery?.data?.balancesMap[Synths.sUSD]?.balance ?? wei(0);
+
+	const stakingInfo = useMemo(() => {
+		const calculatedTargetBurn = Math.max(debtBalance.sub(issuableSynths).toNumber(), 0);
+
+		let amountToBurnBN = parseSafeWei(amountToBurn, 0);
```

Comment:
lol I know this is how I left it after refactoring but maybe its time to rename this from `amountToBurnBN`

also why do you use `, 0)` argument here? This specifies the number of decimals, and if you are burning an amount it would be 18 decimals. It should work best without the extra parameter since it defaults to 18.

Reply:
looks like the second param is actually just the default value to return if the value is invalid https://github.com/Synthetixio/staking/blob/dev/utils/parse.ts#L3